### DISCUSSION TITLE
slug as part of the URL?

### DISCUSSION THREAD

**Discussion by pochat** [nuxt/content#2486] [Q&A]
Hi all, 

I hope you are well.

In the .md files, we can define variables in the front matter like:

---
title: 'my italian trip'
description: 'this is a page about my trip to Venice'
slug: 'venice-trip'
---

It is possible to have the name of our file like blog.md but render the slug in the browser?

For example,
blog1.md renders https://www.example.com/blog/blog1.md

Can we do this?
blog1.md renders https://www.example.com/blog/venice-trip

If so, how do I do it? Would anyone have an example of their code?

Thank you for your time, and have a Nuxt day. :)

**Comment by nobkd** [comment]
I haven't tested it, but the following might work.
You can try setting `_path: /my/new/path` in the front-matter: See https://github.com/nuxt/content/issues/1383#issuecomment-1196237686

I think the `_path` needs to be an absolute path, but I'm not so sure about that.

**Comment by pochat** [comment]
Hi @nobkd,

Thank you for taking the time to respond to my post. I'll review the links you sent and try their solution.

Have a great day.
